stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Channels.java

```diff
@@ -582,14 +581,6 @@ public void channelMute() {
                     List<ChannelMute> mutes = result.data().getUser().getChannelMutes();
                 }
             });
-
-            // Get updates about muted channels
-            client.subscribeFor(
-                    new Class[]{ChannelsMuteEvent.class},
-                    channelsMuteEvent -> {
```

Comment:
Shouldn't we subscribe for `notification.channel_mutes_updated` and/or `notification.mutes_updated` instead? :thinking:

Reply:
Updated to `NotificationChannelMutesUpdatedEvent` event.